docker/templates/killbill/latest/Dockerfile

```diff
@@ -25,4 +26,4 @@ ENV KPM_INSTALL_CMD ansible-playbook $ANSIBLE_OPTS  \
                                      -e catalina_base=$CATALINA_BASE \
                                      $KILLBILL_CLOUD_ANSIBLE_ROLES/killbill.yml
 # Run kpm install
-CMD ["bash", "-c", "$KPM_INSTALL_CMD && $START_TOMCAT_CMD && tail -F $CATALINA_BASE/logs/killbill.out"]
\ No newline at end of file
+CMD ["bash", "-c", "$KPM_INSTALL_CMD && $START_TOMCAT_CMD && tail -F $CATALINA_BASE/logs/catalina.out"]
```

Comment:
We want `killbill.out` instead (may not exist right away, hence the `-F` option).

Reply:
Got it, I will change it back.  The reason of the change was because the container never continue beyond that point (Windows with docker toolbox - Docker machine with Virtual Box).

Reply:
Weird, `-F` is `-f --retry`, i.e. it should keep trying to open the file if it is inaccessible.

Reply:
Maybe a safer option would be to `touch` it first...